tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "moduleResolution": "Node",
 
     "allowSyntheticDefaultImports": true,
+    "declaration": true,
```

Comment:
This will make it easier for people to extend TS-ESNode in their own scripts. Recently realized that most of my modules don't have declaration enabled, oops.